system-parachains/encointer/src/lib.rs

```diff
@@ -0,0 +1,953 @@
+pub use pallet_encointer_communities::Call as EncointerCommunitiesCall;
+// pub use pallet_encointer_personhood_oracle::Call as EncointerPersonhoodOracleCall;
+pub use pallet_encointer_faucet::Call as EncointerFaucetCall;
+pub use pallet_encointer_reputation_commitments::Call as EncointerReputationCommitmentsCall;
+pub use pallet_encointer_scheduler::Call as EncointerSchedulerCall;
+// pub use pallet_encointer_sybil_gate_template::Call as EncointerSybilGateCall;
+
+pub use encointer_primitives::{
+	balances::{BalanceEntry, BalanceType, Demurrage},
+	bazaar::{BusinessData, BusinessIdentifier, OfferingData},
+	ceremonies::{AggregatedAccountData, CeremonyIndexType, CeremonyInfo, CommunityReputation},
+	common::PalletString,
+	communities::{CommunityIdentifier, Location},
+	scheduler::CeremonyPhaseType,
+};
+use sp_core::ConstU32;
+
+// XCM imports
+// Polkadot imports
+use pallet_xcm::{EnsureXcm, IsMajorityOfBody};
+use polkadot_runtime_common::{BlockHashCount, SlowAdjustingFeeUpdate};
+use xcm::latest::BodyId;
+use xcm_executor::XcmExecutor;
+
+// adopt all currency-related constants identical to other system chains
+use deal_with_fees::FeesToTreasury;
+use parachains_common::kusama::{currency::*, fee::WeightToFee};
+use weights::{BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight};
+
+/// A type to hold UTC unix epoch [ms]
+pub type Moment = u64;
+pub const ONE_DAY: Moment = 86_400_000;
+
+/// Maximum number of blocks simultaneously accepted by the Runtime, not yet included
+/// into the relay chain.
+const UNINCLUDED_SEGMENT_CAPACITY: u32 = 1;
+/// How many parachain blocks are processed by the relay chain per parent. Limits the
+/// number of blocks authored per slot.
+const BLOCK_PROCESSING_VELOCITY: u32 = 1;
+/// This determines the average expected block time that we are targeting.
+/// Blocks will be produced at a minimum duration defined by `SLOT_DURATION`.
+/// `SLOT_DURATION` is picked up by `pallet_timestamp` which is in turn picked
+/// up by `pallet_aura` to implement `fn slot_duration()`.
+///
+/// Change this to adjust the block time.
+pub const MILLISECS_PER_BLOCK: u64 = 12000;
+pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
+
+pub type AssetId = AssetIdOf<Runtime>;
+pub type AssetBalance = AssetBalanceOf<Runtime>;
+
+impl_opaque_keys! {
+	pub struct SessionKeys {
+		pub aura: Aura,
+	}
+}
+
+/// This runtime version.
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("encointer-parachain"),
+	impl_name: create_runtime_str!("encointer-parachain"),
+	authoring_version: 1,
+	spec_version: 16,
+	impl_version: 1,
+	apis: RUNTIME_API_VERSIONS,
+	transaction_version: 3,
+	state_version: 0,
+};
+
+/// The version information used to identify this runtime when compiled natively.
+#[cfg(feature = "std")]
+pub fn native_version() -> NativeVersion {
+	NativeVersion { runtime_version: VERSION, can_author_with: Default::default() }
+}
+
+//TODO add meaningful values
```

Comment:
is there a tracking issue for this?

Reply:
It's rather a codebase policy (inherited from Polkadot-SDK) that every TODO must link to its tracking issue.

Reply:
noted. 
commit 4a0d2c2 directly fixes this todo